requirements.txt

```diff
@@ -1,7 +1,7 @@
 asyncpg
 backoff
 databases
-discord.py
+git+https://github.com/Rapptz/discord.py.git@efe81a67fb55f9f2a67bb84810be729ed8f09bc3
```

Comment:
any idea if this has been merged yet?

Reply:
yes, it should be in the 2.4.0 tagged release